builder/build/build-ubi-dependencies.sh

```diff
@@ -14,6 +14,7 @@ export LICENSE_DIR="/THIRD_PARTY_NOTICES"
 mkdir -p "${LICENSE_DIR}"
 
 ### Dependencies
+builder/install/20-googletest.sh
```

Comment:
I suppose this would require changes in `20-googletest.sh` as well.

Reply:
I agree that this is not as elegant as it could be. However, the googletest script here is the first one to be converted to use submodules for both ubi and builder. In the subsequent PR, the `build-ubi-dependencies.sh` is removed in favor of a common script.